### PR TITLE
Add gists in their own folder & creates a markdown file.

### PR DESCRIPTION
**Pull request by joseribes**
Currently there are three gist in my gist section in github. 
I copied them and set in a folder as part of the repo. User can download them. 
Also I've created a file written with markdown with links to them in my gist section.

Closes bigmlcom/bigml-csharp#21

### CODE REVIEW

**Comment by joseribes** [comment]
This changes are about issue #21

File: Samples.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Examples / Gists
+-----------------
+
+You can view some code examples in the Gists folder or in these links below:
+
+
+### PredictionWithIrisEnsemble
+{% gist 97a5b9cff4b17469ece4 %}
```
Comment (osroca) [inline Samples.md]: The way for define links in markdown at Github is:

```
[Create a BigML Prediction using an exinting Ensemble](https://gist.github.com/joseribes/97a5b9cff4b17469ece4)
```

**Comment by osroca** [comment]
Just a little nit before merge it.
The examples look good but we need more! the first one should be one that creates a regression model/ensemble by uploading a new source and then predicts with it. (if it is a model, prediction with the local instance of the model would be nicer).

**Comment by joseribes** [comment]
Please @osroca,  update this. I'm waiting fout days.

**Pull request merged**